dist/less/saas-list.less

```diff
@@ -26,10 +32,12 @@
       @media (min-width: @screen-sm-min) {
         border-bottom: none;
         flex: 0 1 auto;
```

Comment:
Then set flex shrink to 0 (flex: 1 1 auto).
![screen shot 2017-05-17 at 4 53 42 pm](https://cloud.githubusercontent.com/assets/895728/26175675/6b753eaa-3b21-11e7-9bb4-8090796bf960.PNG)

Point is, you shouldn't need to set both width and min-width with flex (assuming you set flex-grow and flex-shrink properly)